its/target-platforms/ibuilds.target

```diff
@@ -16,6 +16,10 @@
       <repository location="https://download.eclipse.org/reddeer/releases/latest" />
       <unit id="org.eclipse.reddeer.eclipse.feature.feature.group" version="0.0.0" />
     </location>
+    <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+      <unit id="org.python.pydev.feature.feature.group" version="0.0.0" />
+      <repository location="http://pydev.sourceforge.net/pydev_update_site/11.0.3" />
```

Comment:
Why is this `pydev_update_site/11.0.3` here and in other places it's `pydev_update_site/10.2.1`?

Reply:
We have 3 target platforms for ITs: **Oldest**, **Latest**, **iBuilds**.

We use **Latest** also in Eclipse for development, **iBuilds** is what is the latest on Eclipse IDE and the plug-in side as we normally don't build with what is either just released or still in development. **Oldest** is of course the oldest stuff we support (or try to support).

Reply:
Make sense, thanks 👍🏻